CODE_REVIEW_CHECKLIST.md

```diff
@@ -69,3 +71,10 @@ may have.
 
 - [ ] Will this change work on all of our supported platforms and
```

Comment:
To be honest, I have no idea about the browsers! 😕 That makes this a bad checklist item. 

*But* since the web-accessible part of _Chef Server_ is quite minimal (oc_id), and should stay that way, this item doesn't say much anyways... 🤔